packages/Directory.Build.targets

```diff
@@ -0,0 +1,2 @@
+<Project>
```

Comment:
Is this needed?

Reply:
It stops the build from traversing up to your build targets.  I'm sure that when you common up the build properties, you will be able to eliminate it.